src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -106,7 +112,25 @@
     @Argument(doc = "Stop after writing this many sequences.  For testing.")
     public int NUM_SEQUENCES = Integer.MAX_VALUE;
 
+    @Argument(shortName = "AN", doc = "Optional file containing the alternative names for the contigs. "
```

Comment:
Explain how this input will be used.

Reply:
To be clear, I mean in the documentation...something like: "The alternative names will be put into the appropriate `@AN` annotation for each contig". Also, please state how the "columns" need to be separated ("tab", I presume) and whether the columns should have a "header" (I presume that they do not)

Reply:
@yfarjoun I added the sentences.